crates/manifest/src/lib.rs

```diff
@@ -65,6 +65,8 @@ pub struct CoreComponent {
     /// ID of the component. Used at runtime to select between
     /// multiple components of the same application.
     pub id: String,
+    /// Description of the component(Optional).
```

Comment:
Suggest removing `optional` here - it's already implicit in the `Option` type. 
```suggestion
    /// Description of the component.
```

Reply:
Make sense. Resolved.